test/integration/001_simple_copy_test/models-merge-update/incremental_update_cols.sql

```diff
@@ -0,0 +1,17 @@
+{{
+    config(
+        materialized = "incremental",
+        unique_key = "id",
+        merge_update_columns = ["email", "ip_address"]
```

Comment:
I'm hopeful that this would "just work" with the recommended changes above. If it doesn't, we may need to try this on Snowflake:
```
        merge_update_columns = ["EMAIL", "IP_ADDRESS"]
```
That's not fun, so I'm hoping we can avoid it.